### DISCUSSION TITLE
법정점검 게시글 조회 API 더미데이터 요청드립니다

### DISCUSSION THREAD

**Discussion by yerim08** [10aeat/10aeat_Backend#169] [FE -> BE 요청사항]
유지보수와 마찬가지로 법정점검 게시글 조회 관련 API들 더미데이터도 요청드립니다!

**Comment by Galmaeki** [comment]
유지보수쪽 문제 해결하고 더미데이터 작성하겠슴다
데이터 넣은뒤에 알려드리겠슴다!

**Comment by yerim08** [comment]
네 ! 천천히 해주세요

**Comment by Galmaeki** [comment]
임시데이터 작성은 했는데 현 DB가 업데이트 이전 DB라 한 게시글에 이슈가 2개이상 작성이 안되는 상태임다!
나머지 테스트는 현재도 가능합니다
고치려면 더미데이터를 날려야해서 이부분은 내일 오전에 수정하면서 더미데이터 다시 넣도록하겠슴다
